docker/build-project.sh

```diff
@@ -34,7 +34,23 @@ set +u
 source refkit-init-build-env ${BUILD_DIR}
 set -u
 
+# Take local CI preferences if present, 1st time, to get bb_e_env parsed
+if [ -f $WORKSPACE/meta-*/conf/distro/include/refkit-ci.inc ]; then
+  cat $WORKSPACE/meta-*/conf/distro/include/refkit-ci.inc > conf/auto.conf
+fi
+
+# use bitbake -e for variables parsing, then pick REFKIT_CI part
+bitbake -e >bb_e_out 2>bb_e_err || (cat bb_e_err && false)
+grep -E "^REFKIT_CI" bb_e_out > ${WORKSPACE}/refkit_ci_vars || true
+
 if [ ! -z ${JOB_NAME+x} ]; then
+  # in CI: run pre-build oe-selftests using development-mode settings in auto.conf
+  echo "include conf/distro/include/refkit-development.inc" > ${WORKSPACE}/build/conf/auto.conf
+  for _test in `grep REFKIT_CI_PREBUILD_SELFTESTS ${WORKSPACE}/refkit_ci_vars | perl -pe 's/.+="(.*)"/\1/g; s/[^ .a-zA-Z0-9_-]//g'`; do
+      oe-selftest --run-tests ${_test}
```

Comment:
I find it nicer to invoke oe-selftest only once, with the entire set of tests as parameter.